release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2476,6 +2476,152 @@ index a90ab14ff..f7f6f1a71 100644
+
+* If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
+
+* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the state does not change (`AtLatestKnown`). (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
+
+* If SRIOV-FEC policies are added separately, the {cgu-operator} adds the policies to the top of the policy list. The original content in the `/source-crs` folder overrides the contents updated by the enforce policy, which causes the original inform policy to never become `Compliant`. The workaround is to add the `CatalogSource` change in the `common-config-policy`. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
+
+* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])
+
+* When you pull the `ztp-site-generate:v4.10.0-53` GitOps ZTP container and attempt to patch the ArgoCD resource, the patch points to a different ztp-site-generate image (`registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.10.0`). (link:https://issues.redhat.com/browse/OCPBUGSM-44261[*OCPBUGSM-44261*])
+
+* Installing {product-title} 4.10.13 on Hewlett Packard Enterprise (HPE) nodes fails. Redfish hardware cannot be inspected due to a Redfish set boot device failure. (link:https://issues.redhat.com/browse/OCPBUGSM-44151[*OCPBUGSM-44151*])
+
+*  Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription for ZT systems servers. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:
++
+[source,terminal]
+----
+$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
+-H 'Content-Type: application/json' \
+--data-raw '{
+"Protocol": "Redfish",
+"Context": "any string is valid",
+"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
+"EventTypes": ["Alert"]
```

Comment:
Remove this line to make the workaround more general since some older ZT system does not recognize the field `EventTypes`.
```
"EventTypes": ["Alert"]
```